kratos/geometries/triangle_3d_3.h

```diff
@@ -421,8 +424,9 @@ template<class TPointType> class Triangle3D3
      * :TODO: could be replaced by something more suitable
      * (comment by janosch)
      */
-    virtual double DomainSize() const override {
-      return Area();
+    virtual double DomainSize() const override 
+    {
+        return Area();

```

Comment:
This is THE example of how a bad name for a method can mislead everyone. DomainSize returns area for 2d shapes and volume for 3d shapes